<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EYE OF GOD  Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/icon/resource/icon/Icon.ico</normaloff>:/icon/resource/icon/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(10, 10, 10);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 127);
border-radius: 5px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="styleSheet">
             <string notr="true">color: rgb(35, 35, 35);</string>
            </property>
            <property name="text">
             <string>Error !!!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_close_pup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-radius: 5px;
	background-image: url(C:/Users/idris/Documents/qt/eye_of_god/resource/images/cil-x.png);
	background-position: center;
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {
	background-color: rgb(50, 50, 50);
	color: rgb(200, 200, 200);
}
QPushButton:pressed {
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="contenu">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="login_area">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>550</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(20, 20, 20);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="logo_esprit">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>-50</y>
            <width>298</width>
            <height>201</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">	background-image: url(C:/Users/idris/Documents/qt/eye_of_god/resource/images/esprit.png);
	background-repeat: no-repeat;
	background-position: center;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="avatar">
          <property name="geometry">
           <rect>
            <x>165</x>
            <y>150</y>
            <width>120</width>
            <height>120</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {
	background-image: url(C:/Users/idris/Documents/qt/eye_of_god/resource/images/avatar.png);
	border-radius: 60px;
	border: 10px solid rgb(255, 0, 0);
}
QFrame:hover {
	;
	border: 10px solid rgb(255, 0, 17);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_login">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>300</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45, 45, 45);
	border-radius:5px;
	padding: 10px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}
QLineEdit:hover{
	border: 2px solid rgb(55, 55, 55);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 0, 0);
	color: rgb(200, 200, 205);
}</string>
          </property>
          <property name="maxLength">
           <number>50</number>
          </property>
          <property name="placeholderText">
           <string>USER</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>355</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45, 45, 45);
	border-radius:5px;
	padding: 10px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}
QLineEdit:hover{
	border: 2px solid rgb(55, 55, 55);
}
QLineEdit:focus{
	border: 2px solid rgb(255, 0, 0);
	color: rgb(200, 200, 205);
}</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>PASSWORD</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>420</y>
            <width>281</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator{
	border: 3px solid rgb(100.100.100);
	width: 15px;
	height: 15px;
	border-radius: 10px;
	background-color: rgb(135, 135, 135);
}
QCheckBox::indicator:checked{
	border: 3px solid rgb(255.0.17);
	background-color: rgb(255, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Se souvenir de moi</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_valider">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>460</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);

}
QPushButton:pressed{
	background-color: rgb(255, 0, 0);
	border: 2px solid rgb(255, 0, 127);
	color: rgb(35, 35, 35);

}</string>
          </property>
          <property name="text">
           <string>CONNECT TO   E Y E  O F  G O D</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(15, 15, 15);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_created">
         <property name="font">
          <font>
           <family>Segoe UI</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(75, 75, 75);</string>
         </property>
         <property name="text">
          <string>Created by: INGENIEUR2.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_close_pup</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
